qutebrowser/config/configdata.yml

```diff
@@ -1030,6 +1030,42 @@ input.spatial_navigation:
     Right key, heuristics determine whether there is an element he might be
     trying to reach towards the right and which element he probably wants.
 
+input.blur_on_load.enabled:
```

Comment:
Maybe those four settings should be in a `input.focus` group (`input.focus.blur_on_load_enabled` etc.)? @mschilli87 what do you think?